src/main/java/org/humancellatlas/ingest/core/web/MetadataController.java

```diff
@@ -47,7 +47,7 @@
         return ResponseEntity.accepted().body(assembler.toFullResource(metadataDocument));
     }
 
-    @RequestMapping(path = "/{metadataType}/{id}" + Links.VALIDATING_URL, method = RequestMethod.PUT)
+    @RequestMapping(path = "/{metadataType}/{id}" + Links.METADATA_VALIDATING_URL, method = RequestMethod.PUT)
```

Comment:
Could this be an `@PutMapping`?

Reply:
Oh I actually didn't touch this.  I think the changes I pulled appeared as my changes.

Reply:
I applied them anyway to all request mappings in the SubmissionController and MetadataController.